.cirrus.default.yml

```diff
@@ -229,11 +222,16 @@ qa_task:
     echo 'Recording tests on video'
     ffmpeg -loglevel warning -f x11grab -video_size 1920x1080 -i ${DISPLAY} -codec:v libx264 -r 12 ${CIRRUS_WORKING_DIR}/recording_${QA_CATEGORY}.mp4
   run_its_script: |
-    echo "Run Maven ITs for Eclipse ${TARGET_PLATFORM} and Server ${SQ_VERSION}"
-    mvn -B -e -V org.jacoco:jacoco-maven-plugin:prepare-agent verify -f its/pom.xml -Pcoverage \
-      -Dtarget.platform=${TARGET_PLATFORM} -Dtycho.localArtifacts=ignore -Dsonarlint-eclipse.p2.url="file://${CIRRUS_WORKING_DIR}/staged-repository" -Dsonar.runtimeVersion=${SQ_VERSION} \
+    echo "Run Maven ITs for Connected Mode with SonarQube on Eclipse 2024-03 / 4.31 (latest Java 17) and Server ${SQ_VERSION}"
+    cd its/
+    mvn -B -e -V org.jacoco:jacoco-maven-plugin:prepare-agent verify \
+      -P coverage,\!standaloneMode,\!connectedModeSc \
```

Comment:
`\!standaloneMode,\!connectedModeSc` what does this mean?

Reply:
Disable the profiles, this is the black magic I was talking about. In order to work correctly with Maven + Tycho + inside the Eclipse IDE we have to have everything available and enabled, but when we run on the CI/CD we cannot run all the tests together and will therefore also skip the modules completely (compilation and so on).